### ISSUE TITLE
Using ng-if causes angular $animate error

### ISSUE THREAD

**Issue by DrClick** [Famous/famous-angular#300]
if you use an ng-if with famous angular, this error is thrown: 
Cannot read property 'then' of undefined
    at Object.ngIfWatchAction [as fn] (http://localhost:8000/lib/angular/angular.js:23358:47)
    at Scope.$digest (http://localhost:8000/lib/angular/angular.js:14133:29)
    at Scope.$apply (http://localhost:8000/lib/angular/angular.js:14395:24)
    at done (http://localhost:8000/lib/angular/angular.js:9569:47)
    at completeRequest (http://localhost:8000/lib/angular/angular.js:9756:7)
    at XMLHttpRequest.requestLoaded (http://localhost:8000/lib/angular/angular.js:9697:9)

**Comment by ghaiat** [comment]
Hey @zackbrown 
Any update? i am facing this issue any time i am using an ng-if

**Comment by zackbrown** [comment]
Hey @ghaiat — I just went ahead and put together a codepen template for F/A:  http://codepen.io/zackbrown/pen/yyVQje (I put the current master onto the CDN preemptively as 0.6.0, though that CDN endpoint is subject to change until the release is actually cut.)

I also put together one that uses `ng-if` and it seems to be working correctly.  http://codepen.io/zackbrown/pen/ogYQyq .  I tried the ng-if on a surface, on its parent modifier, and on the content of the surface (in this case, an `img` tag) and none of them threw errors.  Are either you or @DrClick able to reproduce your error scenarios by modifying that codepen?
